chevah/compat
# [#3437] Add code coverage reporting via codecov.

Scope
=====

This add support for coverage reporting via codecov.

I hope that compat is pretty small and codecov can handle it

As a start, I want to use the coverage to make sure that we have 100% for our test code. This will make sure that all testing code is executed and we don't have tests code which we think is executed, but is not.


change
======

update build system

as a drive by I added the github template and paver.sh

How to test
=========

reviewers: @alibotean @brunogola 

to have some diff, I have pushed my local coverage results as being from master.

https://codecov.io/gh/chevah/compat/pull/59?src=pr

check that changes make sense

```
paver test_review 59
```
